sky/clouds/service_catalog/__init__.py

```diff
@@ -354,6 +354,19 @@ def get_image_id_from_tag(tag: str,
     return _map_clouds_catalog(clouds, 'get_image_id_from_tag', tag, region)
 
 
+def get_image_os_from_tag(tag: str,
+                          region: Optional[str] = None,
+                          clouds: CloudFilter = None) -> Optional[str]:
+    """Returns the image OS type from the tag. OS type info would be useful
+    if different OS type is supported to provision VM, such as ssh username,
+    setup commands, etc.
+    """
+    try:
+        return _map_clouds_catalog(clouds, 'get_image_os_from_tag', tag, region)
+    except AttributeError:
+        return None
```

Comment:
This adds a new API to service_catalog although it is only used for oci catalog. How about instead, we let `oci.py` to call the underlying `oci_catalog.py`'s function directly for this?

Reply:
Actually i considered this as well. OS should be an basic parameter for an image, from this point of view, it might be useful by other vendors. But for the time being, it looks only neccessary for oci. So i have no clear idea about it, so please suggest because you are mastering the overall modules of the whole. :)